api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImplTest.java

```diff
@@ -220,8 +220,8 @@ public void searchForObs_shouldSupportMappedAndUnmappedConcepts() {
 	@Test
```

Comment:
I appreciate the initiative, but unless these are directly necessary for the encounter stuff, don't modify these tests here. I've got [another ticket](https://issues.openmrs.org/browse/FM2-89) to deal with that.

Reply:
Can we actually remove this file from the PR?